tools/dependencies.props

```diff
@@ -1,15 +1,12 @@
 <Project>
   <PropertyGroup>
-    <RuntimeFrameworkVersion>2.1.0-preview2-25610-02</RuntimeFrameworkVersion>
     <CoreFxVersion>4.3.0</CoreFxVersion>
     <RoslynVersion>2.6.0-rdonly-ref-61915-01</RoslynVersion>
-    <SystemMemoryVersion>4.5.0-preview2-25610-02</SystemMemoryVersion>
-    <SystemCompilerServicesUnsafeVersion>4.5.0-preview2-25610-02</SystemCompilerServicesUnsafeVersion>
-    <SystemNumericsVectorsVersion>4.5.0-preview2-25610-02</SystemNumericsVectorsVersion>
+    <SystemMemoryVersion>4.4.0-preview2-25405-01</SystemMemoryVersion>
```

Comment:
@muratg, just confirmed, we are referencing the right NuGet package version here.

Reply:
Awesome, thanks for checking!